addons/mail/static/src/discuss/call/common/call_menu.js

```diff
@@ -9,6 +9,38 @@ export class CallMenu extends Component {
     setup() {
         super.setup();
         this.rtc = useState(useService("discuss.rtc"));
+        this.state = useState({
+            systrayIcon: undefined,
+        });
+        this.lastActions = [];
+        this.actionsIcons = [
+            "fa-desktop",
+            "fa-video-camera",
+            "fa-hand-paper-o",
+            "fa-deaf",
+            "fa-microphone-slash",
+        ];
+        useEffect(
+            (...actions) => {
+                for (let i = 0; i < actions.length; i++) {
+                    if (actions[i] !== this.lastActions[i]) {
+                        this.state.systrayIcon = actions[i]
+                            ? this.actionsIcons[i]
+                            : this.actionsIcons[actions.findIndex((v) => v)];
+                        break;
+                    }
+                }
+
+                this.lastActions = actions;
+            },
+            () => [
+                this.rtc.state.sendScreen,
+                this.rtc.state.sendCamera,
+                this.rtc.selfSession?.raisingHand,
+                this.rtc.selfSession?.isDeaf,
+                this.rtc.selfSession?.isSelfMuted,
```

Comment:
For consistency, pick everything from `selfSession`: `selfSession?.isCameraOn` and `selfSession?.isScreenSharingOn`